card-app/src/getUsers.js

```diff
@@ -0,0 +1,19 @@
+import { UserCard } from "./UserCard";
+
+async function getUsers(userID) {
+    
+    const url = `https://reqres.in/api/users/${userID}`;
```

Comment:
👍 👍 - nice work giving this variable a name! Small thing, but it makes this way easier to read.